components/Tooltips/User/UserPopover.tsx

```diff
@@ -0,0 +1,219 @@
+import { useEffect, useState } from "react";
+import { css, StyleSheet } from "aphrodite";
+import { generateApiUrl } from "~/config/api";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faGraduationCap } from "@fortawesome/pro-solid-svg-icons";
+import Image from "next/image";
+
+// Components
+import Button from "../../Form/Button";
+
+// Utils
+import { createEditorSummary, createEduSummary } from "~/config/utils/user";
+import { RHUser, parseUser, ID } from "~/config/types/root_types";
+import { timeSince } from "~/config/utils/dates";
+import colors from "~/config/themes/colors";
+
+const TRUNCATE_SIZE = 100;
+
+const UserPopover = ({ userId }: { userId: ID }) => {
+  const [fetchedUser, setUser] = useState<RHUser | null>(null);
+  const [loading, setLoading] = useState(true);
+  const [showMore, setShowMore] = useState(false);
+
+  useEffect(() => {
+    const fetchPopover = async () => {
+      const url = generateApiUrl(`popover/${userId}/get_user`);
+      const resp = await fetch(url);
+      const json = await resp.json();
+      if (resp.ok) {
+        setUser(parseUser(json));
+        setLoading(false);
+      }
+    };
+
+    fetchPopover();
+  }, []);
+
+  if (loading) {
+    return null;
+  }
+
+  const educationSummary = createEduSummary(fetchedUser?.authorProfile);
+
+  return (
+    <div className={css(styles.container)}>
+      <div
+        style={{
+          display: "flex",
+          alignItems: "flex-start",
+          justifyContent: "flex-start",
+        }}
+      >
+        <Image
+          height={56}
+          width={56}
+          className={css(styles.avatar)}
+          alt={`${fetchedUser?.firstName} ${fetchedUser?.lastName}`}
+          src={fetchedUser?.authorProfile?.profileImage || ""}
+        />
+        <div className={css(styles.descSection)}>
+          <div className={css(styles.name)}>
+            {fetchedUser?.firstName} {fetchedUser?.lastName}
+          </div>
+
+          <div
+            className={css(
+              styles.desc,
+              !fetchedUser?.authorProfile?.description && styles.autoDesc
+            )}
+          >
+            {fetchedUser?.authorProfile?.description
+              ? !showMore
+                ? fetchedUser?.authorProfile?.description.slice(
+                    0,
+                    TRUNCATE_SIZE
```

Comment:
Can we use `truncateText` function? We use it in various place in the codebase.

https://github.com/ResearchHub/researchhub-web/blob/b6977730130f80f043dbf0014007aaeea69ef7f5/config/utils/string.js#L19